packages/core-components/src/components/ProgressBars/GaugeCard.tsx

```diff
@@ -26,6 +26,8 @@ type Props = {
   variant?: InfoCardVariants;
   /** Progress in % specified as decimal, e.g. "0.23" */
   progress: number;
+  hoverMessage?: ReactNode;
+  iconInfoMessage?: string;
```

Comment:
Ah, hmmmm. What if there was a single `icon?: ReactNode` prop, and the user passed in `icon={<Tooltip title="hello"><SomeIconThatIChoose /></Tooltip>}`? That leaves full control in the hands of the implementer, has the most open possible interface, and almost zero implementation cost in this component?